CHANGELOG.md

```diff
@@ -33,6 +33,20 @@ end
 
 You can also use `TestProf::FactoryDefault.disable!(&block)`.
 
+- Add support for global `before_all` tags ([@maxshend][])
```

Comment:
> Could you please add a change log entry?

@palkan I think I've already added a log entry. Do I need to change anything else ? Thank you

Reply:
I'm sorry, confused it with documentation update.

All good 👍